tests/integration/source_code/test_jobs.py

```diff
@@ -47,6 +46,34 @@ def test_job_linter_no_problems(simple_ctx, ws, make_job):
     assert len(problems) == 0
 
 
+def test_job_task_linter_no_problems(simple_ctx, ws, make_job, make_random, make_cluster, make_notebook):
+    created_cluster = make_cluster(single_node=True)
+    entrypoint = WorkspacePath(ws, f"~/linter-{make_random(4)}").expanduser()
+    entrypoint.mkdir()
+
+    notebook = entrypoint / "notebook.ipynb"
+    task = jobs.Task(
+        task_key=make_random(4),
+        description=make_random(4),
+        existing_cluster_id=created_cluster.cluster_id,
+        notebook_task=jobs.NotebookTask(
+            notebook_path=str(
+                make_notebook(
+                    path=notebook,
+                    content=b"""import doesnotexist
+import greenlet""",
```

Comment:
Not how you should write python, but nicer to read in the test
```suggestion
                    content=b"import doesnotexist; import greenlet",
```